docs/reference/components/transforms/log_to_metric.cue

```diff
@@ -3,6 +3,10 @@ package metadata
 components: transforms: log_to_metric: {
 	title: "Log to Metric"
 
+	description:	"""
+		Derives one or more Vector metric events from a Vector log event.
```

Comment:
Nit: "Vector metric events" and "Vector log event" could lead one to think that they're coming from Vector. Maybe clarify a bit but not blocking.